src/logging.ts

```diff
@@ -25,6 +26,8 @@ export function setupLogging( {
 		app.setAppLogsPath();
 	}
 
+	const isWindows = platform() === 'win32';
```

Comment:
We have a function `isWindows` but it seems it only works in the front-end. So It's ok to make the comparison here.